docs/advanced/cmsis-rtos.md

```diff
@@ -0,0 +1,77 @@
+# CMSIS & RTX
+
+CMSIS/RTX code is imported from the original CMSIS repository which can be found: https://github.com/ARM-software/CMSIS_5/
```

Comment:
CMSIS/RTX is a separate repository (also owned by ARM). We take code from this external repo and import it in mbed OS codebase. Every now and again we will import new version of the code.